roadmap.rb.md

```diff
@@ -0,0 +1,191 @@
+    def self.pin_wait_for2(pin, trigger)
```

Comment:
This seems to be a mistake from me, this shouldn't be checked in.. to be removed